### PR TITLE
Add `detections-vehicles` dataset to backend

### PR DESCRIPTION
**Pull request by leothomas**
### What I did: 
Added `detections-vehicles` dataset - addresses [covid-dashboard issue #440](https://github.com/NASA-IMPACT/covid-dashboard/issues/440)

### How I did it: 
Extracted scene_ids for each feature in the two geojsons currently in S3 and built + uploaded a `detections_scenes.csv` file. 

Created a new dataset metadata file called `detections-vehicles.json` which is very similar to `detection-plane.json` and `detections-ship.json`.

Added `detections-vehicle` to all datasets which has `detections-ship` and `detection-plane` as exclusive datasets

### How you can test it: 
Run `docker-compose up --build` 

- Hit `http://localhost:8000/v1/datasets` notice a new dataset for `detections-vehicles` is present (with `{spotlightId}` in url, since no spotlight has been specified)
- Hit `http://localhost:8000/v1/datasets/la` notice that the `detections-vehicles` is still present
- Hist `http://localhost:8000/v1/datasets/ny` notices that the `detections-vehicles` is no longer present

@olafveerman @drewbo would love some feedback on the `detections-vehicles.json` metadata files (I copy-pasted the description from `detections-ships.json`, not sure if the `info` field is missing anything)

@danielfdsilva tagging you because the format of the geojsons in s3 seems different from the other `detection-*` datasets. Let me know if you need anything in order to implement in the front end

@lillythomas fyi

### CODE REVIEW

**Comment by olafveerman** [comment]
Thanks @leothomas 

* `detections-ships.json` seems to check out. My only comment is that the interface shows detections in orange (not red), I can update the description for all ML detections
* had a quick look at the GeoJSON, and didn't notice any structural difference that would trip up the frontend

Running this with 1.8.0 of the dashboard, I noticed a couple of issues. Unsure whether they should be solved on the API, or frontend, but raising them here:

* background imagery is not available 
* there are vehicle detections for two dates. The domain in the API response is: ` "domain": [ "2020-01-09T00:00:00Z", "2020-07-08T00:00:00Z" ]`. Instead of treating these as two discrete dates, the frontend interprets this as a range daily dates with the domain being the start and end date

**Comment by danielfdsilva** [comment]
Tested this locally and the data seems to show up correctly.
I encountered some errors, but I think it's because I don't have a planet key.

I did notice the domain error reported by @olafveerman, and this is a problem with the frontend. We were considering domains with 2 dates to be a range according to the time unit. This has been the case and worked until this moment. The very useful `isPeriodic` property of the api response was not being used.

Long story short, here's the PR with the fix: https://github.com/NASA-IMPACT/covid-dashboard/pull/455

**Comment by leothomas** [comment]
Fixed the issue with `detections-vehicles` missing background scenes (issue was due to the fact that `detections-vehicles` relies on Skysat imagery, which is not available through the Planet tile endpoint (`detections-plane` and `detections-ship` rely on Planet imagery which _is_ available through the Planet tile endpoint). 

@olafveerman re: observations colour - I'm not seeing anywhere on the backend to specify the colours of the observations. If that's not fixable through the front end, I'm happy to update the `info` field of each of the `detections-*` metadata files to specify that the observations are show in orange (instead of red)

**Comment by olafveerman** [comment]
This looks good! Holding off on merging for a couple of days, until the Story feature has been launched and stable.

**Comment by leothomas** [comment]
@olafveerman This PR has been open for a while. Can we merge, now that the Story feature has been launched?
